docs/conf/cfp/east-2020.rst

```diff
@@ -0,0 +1,40 @@
+Write the Docs East 2020 Call for Venues
+========================================
+
+We're also excited to start planning another US conference in 2020.
+The goal will be to make closer to folks on the east coast,
+but we haven't figured out exactly where to host it yet.
+
+The plan is to host it sometime in the fall,
+so that it is nicely spaced out with our Portland conference.
+
+Requirements
+------------
+
+We have some lose requirements for venues that we're looking for.
+It's OK if a venue doesn't fit one or two of these,
+but they are the general goal of what we're looking for.
+
+* Fits up to 250 people
+    - Preferably with some tables along with chairs, but not a requirement
+* Has character. We are quite unlikely to try and use a conference center or hotel basement, as that is really not the vibe we're going for.
```

Comment:
Maybe change the second sentence to:

We are unlikely to use a conference center or hotel space--it's not the vibe we're going for.